lib/myset.h

```diff
@@ -148,7 +148,7 @@ void set2container(const char* str, Insert insert)
 	}
 
 	// Save final element of set, if any
-	if (temp.size()) {
+	if (!temp.empty()) {
```

Comment:
[Applied](https://github.com/tangentsoft/mysqlpp/commit/89fd6f5051d7ea1baf3d649226844b078854429b). Note that the commit comment explains why it's changing, rather than just sweep it into the vast maw of "C++11 enhancement".